pgraph/graphviz_test.go

```diff
@@ -0,0 +1,103 @@
+// Mgmt
+// Copyright (C) 2013-2019+ James Shubin and the project contributors
+// Written by James Shubin <james@shubin.ca> and the project contributors
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// +build !root
+
+package pgraph
+
+import (
+    "testing"
+//	"fmt"
+//	"io/ioutil"
+)
+
+// based on pgraph_test.go TestAddVertex() - line 54
+func TestGraphViz1(t *testing.T){
+	t.Logf("what up")
+    G := &Graph{Name: "g2"}
+	v1 := NV("v1")
+    v2 := NV("v2")
+    v3 := NV("v3")
+    v4 := NV("v4")
+    v5 := NV("v5")
+    v6 := NV("v6")
+    e1 := NE("e1")
+    e2 := NE("e2")
+    e3 := NE("e3")
+    e4 := NE("e4")
+    e5 := NE("e5")
+    e6 := NE("e6")
+
+	G.AddEdge(v1, v2, e1)
+    G.AddEdge(v2, v3, e2)
+    G.AddEdge(v3, v1, e3)
+
+    G.AddEdge(v4, v5, e4)
+    G.AddEdge(v5, v6, e5)
+	G.AddEdge(v6, v4, e6)
+
+	var expectedResult = [...]string{
+        "digraph "g2" {",
+        "	label=\"g2\";",
+        "	\"0xc00004a510\" [label=\"v1\"];",
```

Comment:
Quiz: What are these 0xc... things?

Reply:
Data addresses.

Reply:
Why would it be bad to have these in a test?

Reply:
Because they change from machine to machine